src/core/k8.mjs

```diff
@@ -33,6 +33,8 @@ import { V1ObjectMeta, V1Secret } from '@kubernetes/client-node'
  * For parallel execution, create separate instances by invoking clone()
  */
 export class K8 {
+  static _webSocketGetters = []
```

Comment:
Would be nice to have some description of the variable. It was not clear what it is.

Reply:
done